src/tests/ftest/nvme/enospace.py

```diff
@@ -353,26 +351,32 @@ def test_performance_storage_full(self):
         # Write First
         self.start_ior_load(storage='SCM', operation="Auto_Write", percent=1)
         # Read the baseline data set
-        self.start_ior_load(storage='SCM', operation='Auto_Read', percent=1)
+        self.start_ior_load(storage='SCM', operation='Auto_Read', percent=1, create_cont=False)
```

Comment:
We only want to increase the number of ior repetitions in two commands - this one:
```suggestion
        self.start_ior_load(storage='SCM', operation='Auto_Read', percent=1, create_cont=False, repetitions=100)
```

Reply:
done.